lib/generators/rolemodel/linters/eslint/templates/eslint.config.js

```diff
@@ -0,0 +1,98 @@
+import globals from 'globals'
+
+export default [
+  {
+    ignores: ['app/assets/builds/**/*']
+  },
+  { name: 'airbnb' },
+  {
+    languageOptions: {
+      parserOptions: {
+        ecmaFeatures: {
+          jsx: true
+        },
+        ecmaVersion: 'latest',
+        globals: {
+          ...globals.browser,
+          ...globals.es6
+        }
+      }
+    },
+    rules: {
+      'no-unused-vars': [
+        'error',
+        {
+          argsIgnorePattern: '^_', // Allow unused arguments starting with an underscore
+          varsIgnorePattern: '^_' // Allow unused variables starting with an underscore
+        }
+      ],
+
+      // No semicolons
+      semi: ['error', 'never'],
+      'no-unexpected-multiline': 'error',
+
+      // 1. We follow the convention of naming "private" properties with a leading underscore
+      // 2. This conflicts with the no-unused-vars config
+      'no-underscore-dangle': 'off',
+
+      // Don't need trailing commas everywhere
+      'comma-dangle': ['error', 'never'],
```

Comment:
This [is deprecated](https://eslint.org/docs/latest/rules/comma-dangle) but I like dangling commas because they reduce git diff churn.

Reply:
For reference, it was pulled from an app which had ESLint 8.52, which marks this as valid